apps/frontend/src/libs/enums/route-access.ts

```diff
@@ -0,0 +1,7 @@
+const RouteAccess = {
+	ALL: "All",
```

Comment:
What is the difference between "All" and "Not authenticated" route accesses? I think we can remove "ALL" and use only "NOT_AUTHENTICATED"

Reply:
Also replace all places in the PR where ALL with NOT_AUTHENTICATED